### PR TITLE
Testnet 19 bph wrapper rename

### PR DESCRIPTION
**Pull request by Ben-PH**
* [x] document all added functions
* [x] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

### CODE REVIEW

File: massa-models/src/secure_share.rs [hunk 1]
```diff
@@ -45,16 +49,17 @@ pub trait Id {
...
-pub trait WrappedContent
+pub trait SecureShareContent
 where
     Self: Sized + Display,
 {
-    /// Creates a wrapped version of the object
-    fn new_wrapped<SC: Serializer<Self>, U: Id>(
+    /// Using the provided key-pair, generates values the are used
+    /// by the trust-free data sharing protocols.
+    fn secure<Ser: Serializer<Self>, ID: Id>(
```
Comment (AurelienFT) [inline massa-models/src/secure_share.rs:58]: Maybe rename it to `new_secure`

**Review by AurelienFT — COMMENTED** [review]
Nice refactor. Just a little adjustment on my side and it would be perfect

File: massa-models/src/secure_share.rs (hunk 1 above)
Comment (Ben-PH) [reply massa-models/src/secure_share.rs:58]: I like it. I'll put that together on Monday.

**Comment by Ben-PH** [comment]
@Leo-Besancon I thought you'd be a valuable review from the perspective that the new naming convention (and some doc-comment tweaks) relate closely to our lessons with damir about how signing and verification works.

**Comment by Leo-Besancon** [comment]
The proposed change does make it clearer to understand the intent of Wrapped types.
Just to be sure, you changed 
```
creator_address: op.creator_address,
```
to: 
```
creator_address: op.content_creator_address,
```

in order to differentiate between the creator of the wrapped content (e.g. the Block producer) and the creator of the wrapper struct?

**Comment by Ben-PH** [comment]
> in order to differentiate between the creator of the wrapped content (e.g. the Block producer) and the creator of the wrapper struct?

That wasn't the intent, and I can now see how this rename is a touch misleading. This value is determined by the keypair that is generated, which is Rng'd, and passed into the creation function. Perhaps `signature_origin_address`? or `signature_public_key_addr`?

I'm trying to go for something that gives an intuitive sense that this is a) a massa address, b) said address is the result of the same public key that is used as the input for generating the signature.

**Comment by Leo-Besancon** [comment]
Understood, content_creator_address is fine then.

**Review by Leo-Besancon — APPROVED** [review]

File: massa-models/src/secure_share.rs (hunk 1 above)
Comment (Ben-PH) [reply massa-models/src/secure_share.rs]: I went with `new_verifiable`. I'm thinking "secure" is a touch redundant to the type name, and it's a chance to document the properties that make it `SecureShare`

**Pull request merged**